wordpress-mobile/gutenberg-mobile
# Use v0.60.0 patch from jtreanor/react-native

This updates gutenberg-mobile to use a version of RN `v0.60.0` which is patched with my changes from https://github.com/facebook/react-native/pull/25393. I have just cherry-picked my changes onto the release `v0.60.0` version so there are no other unexpected changes.

I have updated the podspecs of our dependencies:

- https://github.com/wordpress-mobile/react-native-video/pull/1
- https://github.com/wordpress-mobile/react-native-recyclerview-list/pull/15
- https://github.com/wordpress-mobile/react-native-keyboard-aware-scroll-view/pull/8
- https://github.com/wordpress-mobile/react-native-svg/pull/4

The alternative to the above changes would be to maintain our own copies of the podspecs in `third-party-podspecs`. You can exclude the podspecs from the diff by going [here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1184/files?file-filters%5B%5D=.h&file-filters%5B%5D=.lock&file-filters%5B%5D=.podspec&file-filters%5B%5D=.sh).

~To do: Patch the podspec of `react-native-safe-area` (its manually updated in this PR for now).~ This is now done in a fork here: https://github.com/wordpress-mobile/react-native-safe-area/tree/rn-0.60.0-patch

To test:

- Everything works as normal. There should be no impact to the Gutenberg workflow.
- WPAndroid still works correctly.
- WPiOS works with this change. The corresponding PR is here: https://github.com/wordpress-mobile/WordPress-iOS/pull/12056

Update release notes:
